### ISSUE TITLE
Wont run from AUR with Manjaro

### ISSUE THREAD

**Issue by sumnerevans** [sublime-music/sublime-music-python#106]
In GitLab by @DerekTheGuy on Jan 31, 2020, 15:26

```
Traceback (most recent call last):

  File "/usr/bin/sublime-music", line 11, in <module>
    load_entry_point('sublime-music==0.8.10', 'console_scripts', 'sublime-music')()

  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)

  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()

  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()

  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)

  File "/usr/lib/python3.8/site-packages/sublime/__main__.py", line 10, in <module>
    from .app import SublimeMusicApp

  File "/usr/lib/python3.8/site-packages/sublime/app.py", line 13, in <module>
    from .ui.main import MainWindow

  File "/usr/lib/python3.8/site-packages/sublime/ui/main.py", line 8, in <module>
    from . import albums, artists, browse, playlists, player_controls

  File "/usr/lib/python3.8/site-packages/sublime/ui/albums.py", line 9, in <module>
    from sublime.state_manager import ApplicationState

  File "/usr/lib/python3.8/site-packages/sublime/state_manager.py", line 7, in <module>
    from gi.repository import NetworkManager, NMClient

  File "/usr/lib/python3.8/site-packages/gi/importer.py", line 132, in load_module
    raise ImportError('cannot import name %s, '

ImportError: cannot import name NetworkManager, introspection typelib not found
```

**Comment by sumnerevans** [comment]
@DerekTheGuy thanks for reporting this issue. This should be fixed in the latest AUR version: `0.8.11-1`. Looks like `libnm-glib` and `libnotify` were not included as dependencies in the `PKGBUILD`.

**Comment by sumnerevans** [comment]
closed
